test/WebSites/RazorPagesWebSite/ModelWithPageFilter.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace RazorPagesWebSite
+{
+    [HandlerChangingPageFilter]
+    public class ModelWithPageFilter
+    {
+        public string Message { get; private set; }
+
+        public void OnGet()
+        {
+            Message = $"Hello from {nameof(OnGet)}";
+        }
+
+        public void OnGetEdit()
+        {
+            Message = $"Hello from {nameof(OnGetEdit)}";
+        }
+    }
+
+    [AttributeUsage(AttributeTargets.Class)]
+    public class HandlerChangingPageFilterAttribute : Attribute, IPageFilter
```

Comment:
Did we intentionally choose not to add a `PageFilterAttribute` or was that an oversight?